homeassistant/components/sensibo/sensor.py

```diff
@@ -0,0 +1,131 @@
+"""Sensor platform for Sensibo integration."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ELECTRIC_POTENTIAL_VOLT,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN
+from .coordinator import MotionSensor, SensiboDataUpdateCoordinator
+from .entity import SensiboMotionBaseEntity
+
+
+@dataclass
+class BaseEntityDescriptionMixin:
+    """Mixin for required Sensibo base description keys."""
+
+    value_fn: Callable[[MotionSensor], StateType]
+
+
+@dataclass
+class SensiboSensorEntityDescription(
+    SensorEntityDescription, BaseEntityDescriptionMixin
+):
+    """Describes Sensibo Motion sensor entity."""
+
+
+MOTION_SENSOR_TYPES: tuple[SensiboSensorEntityDescription, ...] = (
+    SensiboSensorEntityDescription(
+        key="rssi",
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        state_class=SensorStateClass.MEASUREMENT,
+        name="rssi",
+        icon="mdi:wifi",
+        value_fn=lambda data: data.rssi,
```

Comment:
```suggestion
        value_fn=lambda data: data.rssi,
        entity_registry_enabled_default=False
```

Usually we disable these by default since they tend to fill up the database and trigger frequent state updates